packages/devtools_app/lib/src/shared/file_import.dart

```diff
@@ -227,6 +227,10 @@ Future<DevToolsJsonFile?> importFileFromPicker({
   final lastModifiedTime = await file.lastModified();
   // TODO(kenz): this will need to be modified if we need to support other file
   // extensions than .json. We will need to return a more generic file type.
+
+  // We need this assert
+  // ignore: unnecessary_null_comparison
+  assert(file.name != null);
```

Comment:
is the proper fix here to unwrap `file.name!` where it is used?

Reply:
We will still get warning, because, file.name is not nullable in latest version. It seems version of file in Google3 is outdated.

Reply:
@elliette can you just re-run the roll script in google3 to pick up the version of file_import.dart that is null-safe?

Reply:
I think I do have the null-safe version, I ran the script ~15 mins ago